misc-scripts/xref_mapping/xref_config.ini

```diff
@@ -1435,14 +1435,14 @@ order           = 49
 priority        = 1
 parser          = done_in_official_naming
 
-[source cint_aniseed_v1::ciona_intestinalis]
+#[source cint_aniseed_v1::ciona_intestinalis]
 # Used by ciona_intestinalis
-name            = cint_aniseed_v1
-download        = Y
-order           = 50
-priority        = 1
-parser          = JGI_ProteinParser
-data_uri        = ftp://ftp.jgi-psf.org/pub/JGI_data/Ciona/v1.0/ciona.prot.fasta.gz
+#name            = cint_aniseed_v1
+#download        = Y
+#order           = 50
+#priority        = 1
+#parser          = JGI_ProteinParser
+#data_uri        = ftp://ftp.jgi-psf.org/pub/JGI_data/Ciona/v1.0/ciona.prot.fasta.gz
 
```

Comment:
I would remove the whole configuration rather than commenting it out
We can always recover it through git history if we ever need it again